packages/overlay/src/overlay-trigger.ts

```diff
@@ -55,104 +48,44 @@ export class OverlayTrigger extends LitElement {
     public disabled = false;
 
     private clickContent?: HTMLElement;
-
     private hoverContent?: HTMLElement;
+    private targetContent?: HTMLElement;
```

Comment:
What sort of context would the mouse events be triggered where this content wasn't available that we've added the gating associated to this for? Might be a good add to comments or documentation.

Reply:
I only added the `targetContent` because I need to pass it through to the `Overlay` and `OverlayStack`. The checks for its existence are to make Typescript happy.